medusa/torrent_checker.py

```diff
@@ -24,6 +24,8 @@
 from medusa import app
 from medusa.clients import torrent
 
+import requests
```

Comment:
Since you're only importing requests for the exception class, maybe use
`from requests import RequestException`